.travis.yml

```diff
@@ -27,7 +27,7 @@ env:
 matrix:
   include:
     - env: RUNTIME=3.6 TOOLKITS="pyqt pyqt5 pyside2 wx"
-    - env: RUNTIME=3.6 TOOLKITS="pyqt5 wx" TRAITS_REQUIRES="^=6.0"
+    - env: RUNTIME=3.6 TOOLKITS="pyqt5 wx" TRAITS_REQUIRES="^=6.1"
```

Comment:
Not sure why we need to test against both the latest version of traits (6.2) and the older version (6.1) for the pyqt5 and wx toolkits. I think we can remove the env where we set `TRAITS_REQUIRES`